sale_prodlot_mrp/mrp.py

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#  licence AGPL version 3 or later
+#  see licence in __openerp__.py or http://www.gnu.org/licenses/agpl-3.0.txt
+#  Copyright (C) 2014 Akretion (http://www.akretion.com).
+#  @author David BEAL <david.beal@akretion.com>
+#
+##############################################################################
+
+from openerp import models, api, fields
+
+
+class MrpProduction(models.Model):
+    """ Purpose to generate manufacturing base on custom product raw material
+    """
+    _inherit = 'mrp.production'
+    _service_product_lst = []
```

Comment:
remove this two dead attribute "_service_product_lst" and "_product_config_dict"